lesson02/activity/gulpfile.js

```diff
@@ -59,7 +59,8 @@ function browserSync() {
             baseDir: './'
         },
         notify: false,
-        browser: "google chrome"
+        browser: "chrome.exe"
```

Comment:
Are you using Windows? I'm trying to figure out why this was necessary for your system but not mine.

Reply:
Update: it has to be chrome.exe to work on Windows, 'google chrome' to work on Mac, and neither works on the other OS. Great job finding and fixing this.